craco.config.js

```diff
@@ -0,0 +1,29 @@
+const { ProvidePlugin } = require('webpack')
+
+module.exports = {
+  webpack: {
+    configure: {
+      resolve: {
+        fallback: {
+          path: require.resolve('path-browserify'),
+          timers: require.resolve('timers-browserify'),
+          fs: require.resolve('browserify-fs'),
+          https: require.resolve('https-browserify'),
+          stream: require.resolve('stream-browserify'),
+          http: require.resolve('stream-http'),
+        },
+      },
+      ignoreWarnings: [/Failed to parse source map/],
+    },
+    plugins: {
+      add: [
+        new ProvidePlugin({
+          process: 'process/browser',
+        }),
+      ],
+    },
+  },
+  eslint: {
+    enable: false,
+  },
```

Comment:
We were previously setting this in an environment variable in our package.json scripts but now we can just include it in the config. ESLint takes too much time to run and outputs too much noise in the console if we run it along with Webpack builds. Continue to just use it in IDE